### PR TITLE
Updated userpage

### PR DESCRIPTION
**Pull request by HarshPanchal26**
Resolve all the conflicts of previous commit [commit(306)](https://github.com/priyankarpal/ProjectsHut/pull/306) .

Check is there is still any is issue let me know .

### CODE REVIEW

**Comment by priyankarpal** [comment]
your package.json file is not updated to it's failing

**Comment by HarshPanchal26** [comment]
I made it updated , Check now .

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -90,8 +90,8 @@ const HomePage = () => {
                   to="ProjectsPage"
                   className="inline-block rounded-md border border-transparent  py-3 px-8 text-center font-bold"
                   style={{
-                    backgroundColor: theme?.button?.buttonBgColor,
-                    color: theme?.button?.buttonTextColor,
+                    backgroundColor: theme.button.buttonBgColor,
```
Comment (priyankarpal) [inline src/pages/HomePage.jsx:93]: Add `?`

```suggestion
                    backgroundColor: theme?.button?.buttonBgColor,
```

File: src/pages/HomePage.jsx [hunk 2]
```diff
@@ -90,8 +90,8 @@ const HomePage = () => {
...
+                    color: theme.button.buttonTextColor,
```
Comment (priyankarpal) [inline src/pages/HomePage.jsx:94]: also here 

```suggestion
                    color: theme?.button?.buttonTextColor,
```

File: src/components/ProjectCard.jsx [hunk 3]
```diff
@@ -11,35 +13,50 @@ function ProjectCard({ gh, link, title, description, tech }) {
...
-      <a
-        href={link}
-        target="_blank"
-        rel="noreferrer"
-        className="inline-flex w-full items-center gap-2 justify-center text-center text-white from-indigo-500 via-purple-500 to-pink-500 bg-gradient-to-r  xl:text-[1rem] md:text-[0.8rem]  rounded-md py-2"
+    <>
+      <article
+        className="border shadow-sm rounded-xl py-5 px-3 mb-3 max-w-md mx-auto sm:m-0  border-gray-700 shadow-slate-700/[.7] transition-all duration-200 hover:scale-95"
+        style={{
+          background: theme.navbar.background,
```
Comment (priyankarpal) [inline src/components/ProjectCard.jsx:20]:
```suggestion
          background: theme?.navbar?.background,
```

File: src/components/ProjectCard.jsx [hunk 4]
```diff
@@ -11,35 +13,50 @@ function ProjectCard({ gh, link, title, description, tech }) {
...
+          color: theme.color,
```
Comment (priyankarpal) [inline src/components/ProjectCard.jsx:21]:
```suggestion
          color: theme?.color,
```

File: src/components/SideMenu.jsx [hunk 5]
```diff
@@ -5,24 +5,35 @@ import IconButton from "@mui/material/IconButton"
...
 export default function SideMenu(props) {
+  const { theme } = useContext(ThemeContext)
   return (
-    <div className=" h-screen bg-gray-900 flex flex-col " id="elements-of-sidebar">
+    <div
+      className=" h-screen flex flex-col "
+      id="elements-of-sidebar"
+      style={{
+        background: theme.background,
+        color: theme.color,
```
Comment (priyankarpal) [inline src/components/SideMenu.jsx:15]:
```suggestion
        background: theme?.background,
        color: theme?.color,
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/pages/HomePage.jsx (hunk 2 above)
Comment (HarshPanchal26) [reply src/pages/HomePage.jsx]: done

File: src/components/ProjectCard.jsx (hunk 3 above)
Comment (HarshPanchal26) [reply src/components/ProjectCard.jsx]: done

File: src/components/ProjectCard.jsx (hunk 4 above)
Comment (HarshPanchal26) [reply src/components/ProjectCard.jsx]: done

File: src/components/SideMenu.jsx (hunk 5 above)
Comment (HarshPanchal26) [reply src/components/SideMenu.jsx]: done

**Comment by HarshPanchal26** [comment]
All done ,I apologies brother , I don't know what happen but I didn't remove any code , but still now it's working fine .

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**